### PR TITLE
first pull for testing

### PR DESCRIPTION
**Pull request by Licini**
1. fixed two test warnings because of python deprecations

2. moved the current test files under `test/compas` folder to mirror the file structure from `src/`, so later we can also have `test/compas_rhino`, `test/compas_blender` etc

1. completed test script for `compas.geometry.bbox` functions

### CODE REVIEW

**Comment by gonzalocasas** [comment]
Most model tests are broken now

File: src/compas/files/urdf.py [hunk 1]
```diff
@@ -217,7 +217,8 @@ def get_metadata(type):
     if hasattr(type, 'from_urdf'):
         metadata['from_urdf'] = getattr(type, 'from_urdf')
     else:
-        argspec = inspect.getargspec(type.__init__)
+        # argspec = inspect.getargspec(type.__init__) # this is deprecated in python3
+        argspec = inspect.getfullargspec(type.__init__)
```
Comment (gonzalocasas) [inline src/compas/files/urdf.py]: I think this doesn't work on python 2

**Review by gonzalocasas — CHANGES_REQUESTED** [review]
The change still needs some work, it breaks on python 2 (python death clock is ticking but is not dead yet) ~, and I'm not sure about the moved files, I don't see that you updated the loading of them so I would assume they are not working either (?) I didn't check this thou, I might be wrong.~

File: src/compas/files/urdf.py (hunk 1 above)
Comment (Licini) [reply src/compas/files/urdf.py]: Hi Gonzalo, thanks for the review. yep, was fully aware we will still use 2.7. Will make change regarding to this part

Comment (Licini) [reply src/compas/files/urdf.py]: Also regarding to moved files,  would you mind to point out what/where is the loading you are referring to? I'm under the impression everything is being detected automatically when put under `test/` folder, and I was able to execute and pass these test after moving(only with 3.7 though). First day working on COMPAS code base, still getting familiar on everything : )

Comment (gonzalocasas) [reply src/compas/files/urdf.py]: You are correct, sorry! I missed the fact that you moved the entirely of the test + fixtures one level deeper.

Comment (Licini) [reply src/compas/files/urdf.py]: added a detector, now it will use old code for python2.7

**Comment by tomvanmele** [comment]
@gonzalocasas all good now?

**Pull request merged**